controller/src/main/java/org/jboss/as/controller/OperationContextImpl.java

```diff
@@ -1213,12 +1213,12 @@ void releaseStepLocks(AbstractOperationContext.Step step) {
                 // because of minor user impatience can release the controller lock while the
                 // container is unsettled. OTOH, if we don't allow interruption, if the
                 // container can't settle (e.g. a broken service is blocking in start()), the operation
-                // will not be cancellable. I (BES 2012/01/24) chose the former as the lesser evil.
+                // will not be cancellable. We chose the late as the lesser evil.
```

Comment:
I'm going to merge this, but I'm going to send up a patch changing these comments, as my impression is right now we are using false because that improves testsuite stability, which isn't a higher priority.